drivers/interrupt_controller/wuc_ite_it8xxx2.c

```diff
@@ -0,0 +1,125 @@
+/*
+ * Copyright (c) 2022 ITE Corporation. All Rights Reserved
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <device.h>
+#include <drivers/interrupt_controller/wuc_ite_it8xxx2.h>
+#include <dt-bindings/interrupt-controller/it8xxx2-wuc.h>
+#include <kernel.h>
+#include <soc.h>
+#include <soc_common.h>
+#include <stdlib.h>
+
+#include <logging/log.h>
+LOG_MODULE_REGISTER(wuc_ite_it8xxx2, LOG_LEVEL_ERR);
```

Comment:
please register a new logging module in Kconfig, don't hardcode levels

Reply:
Done, but I'm not sure the change satisfies your suggestion or not. 
Or could we do like this?
#define LOG_LEVEL       CONFIG_WUC_LOG_LEVEL
LOG_MODULE_REGISTER(wuc_ite_it8xxx2);